statuslight_control/test/statuslight_control/statuslight_test.exs

```diff
@@ -1,15 +1,14 @@
-defmodule StatuslightControl.StatuslightTest do
+defmodule StatuslightControl.StatuslightGenserverTest do
   use ExUnit.Case
-  import ExUnit.CaptureIO
+  alias StatuslightControl.Statuslight
 
-  setup do
-    #Maybe need to do this through supervision instead
-    StatuslightControl.Statuslight.stop
-    StatuslightControl.Statuslight.start_link(IORenderer)
-    :ok
+  test "current_color" do
+    :sys.replace_state(Statuslight, fn _ -> %{color: "green"} end)
```

Comment:
these sys calls let us manipulate the Genserver's state directly, so we don't have to rely on our other defined methods' effects to test this method.

Reply:
👍  Given that these tests were busted, I like the change.  One of my own todos is to come back and use the IORenderer for new (and this time working) side-effect test, but this is good for now.

Reply:
Yeah we can probably test that in a separate test file